src/mainwindow.ui

```diff
@@ -2541,6 +2547,11 @@ p, li { white-space: pre-wrap; }
     <string>Manage dictionaries</string>
    </property>
   </action>
+  <action name="actionasd">
+   <property name="text">
+    <string>asd</string>
```

Comment:
What does that action? :) Is it a placeholder? The way you currently adding it it well need translation.

Reply:
Nah, I didn't even add that. :thinking: 

The one I added was above that. I wonder how it got in there.

Reply:
If it is a dummy-action that will be removed please name it that way and set the "no tranlation" flag on the text 😁

Reply:
Done